src/blog/2023/03/integration-platform-for-edge-computing.md

```diff
@@ -0,0 +1,68 @@
+
+In the manufacturing and industrial automation industry, the focus of the Industrial Internet of Things (IIoT) has been on integrating industrial processes and equipment to enable real-time monitoring, control, and analysis of data. For many use cases, instead of sending all the data to the cloud, the best practice for processing industrial data is to deploy the application to the edge of the network, referred to as edge computing. By processing the data closer to the data source, edge computing has many benefits, including reduced latency, limited downtown, conserving bandwidth, and increased privacy, and security.
+
+
+## How Node-RED fits into IIoT Edge Computing
+
+A key challenge for IIoT edge computing is the wide variety of different hardware platforms, protocols and sources of data and processes. Over the years, the Node-RED community has built [thousands of nodes and flow](https://flows.nodered.org/)s to support a wider range of these sources of data, including support for [Modbus](https://flows.nodered.org/node/node-red-contrib-modbus), [OPC-UA](https://flows.nodered.org/node/node-red-contrib-opcua), [S7](https://flows.nodered.org/node/node-red-contrib-s7), [MQTT](https://cookbook.nodered.org/mqtt/), etc. Node-RED also has nodes for graphics and dashboards to make it trivial to visualize industrial data. 
+
+Node-RED’s visual programming environment makes it accessible to non-developers. Manufacturing, mechanical, and electrical engineers in the factories are typically the domain experts in understanding the existing systems and often lead IIoT initiatives. Node-RED enables these engineers to quickly innovate and create real value for their organizations. This makes it a popular choice for engineers looking to create edge computing solutions.
+
+
+## PLC and IoT Gateway Vendors Embrace Node-RED
+
+PLC and IoT Gateway vendors are at the forefront of promoting edge computing. They see edge computing as a way to modernize hardware in the factory and for remote asset management. PLC and IoT gateways often sit in front of old legacy systems that don’t have the connectivity or compute platform to enable IIoT applications. 
+
+Many of these hardware vendors realized they need an application delivery platform for their devices. Traditional OT hardware vendors often implement proprietary software stacks that are often difficult to use and closed to integrating with other hardware and software. Forward thinking hardware vendors realized having an open platform is the future of their industry and customers have begun to demand more open platforms. Node-RED’s ease of use, open community and open source license provided the solution many of these hardware vendors were looking for. 
+
+
+## The Standard for Edge Computing and PLCs
+
+Today, Node-RED has been adopted by some of the leading PLC and IoT Gateways vendors. The hardware vendor community appears to have standardized on Node-RED as being the edge computing platform for IIoT.
```

Comment:
Should be Gateway rather than Gateways I think.

Reply:
fixed